modules/navier_stokes/doc/content/source/kernels/INSADEnergyWallConvection.md

```diff
@@ -0,0 +1,16 @@
+# INSADEnergyWallConvection
+
+This object adds a term of the incompressible energy equation of the form
```

Comment:
I also thought the same thing about boundary vs volume, but @pbalest has his reasons for wanting it this way. Perhaps I'll let  him answer your question

Reply:
This term is necessary if you want to simulate an HX not modeling the full geometry (all the pipes if the flow is within the pipes or the shell if is outside). If you impose it on a surface your geometry will have to respect the actual geometry (to have the same area) without considering that only the fluid that touch that surface will heat up, when instead at the outlet of the HX the mixing make sure that everything is well mixed therefore it can be well approximated with a heat exchange on a volume instead of a surface.

Reply:
Thanks for your explanation and it makes senes. Just a minor thing to say, could we use a better name than the "Wall", which makes me think of boundary immediately. My two cents :-)

Reply:
Good idea. I'll change the name